luka2220/DNS-Forwarder
# DNS server created and running.

The DNS server was created and can listen to incoming messages on port 8080.

To test the DNS server:
* ` go run main.go start`
* open a separate terminal window while the program is running to run the command below
* `dig @127.0.0.1 -p 8080 www.google.com`

An example output should look like:
![Screenshot 2024-01-16 at 10 00 23 AM](https://github.com/luka2220/DNS-Forwarder/assets/42144047/f3152265-f9be-4954-a5b2-06f6e1d47cf1)
